calcom/cal.com
# Add line breaks, styling, and/or markdown to event description

## What does this PR do?



Fixes #1909  
Also fixes #3348 

## Type of change



## How should this be tested?


- Run `yarn dx` to start application
- Go to `$url/event-types/[type]`
- Check the textarea with the description label

## Demo

[https://www.loom.com/share/e2b6f07525294925ba3feb7fba04a3e1](https://www.loom.com/share/e2b6f07525294925ba3feb7fba04a3e1)
